src/main/resources/assets/computercraft/lua/rom/startup

```diff
@@ -209,22 +209,44 @@ if fs.exists( "/rom/autorun" ) and fs.isDir( "/rom/autorun" ) then
 	end
 end
 
+local function findStartups( sBaseDir )
+    local tStartups = nil
+    local sBasePath = "/" .. fs.combine( sBaseDir, "startup" )
+    local sStartupNode = shell.resolveProgram( sBasePath )
+    if sStartupNode then
+        tStartups = { sStartupNode }
+    end
+    -- It's possible that there is a startup directory and a startup.lua file, so this has to be
+    -- executed even if a file has already been found.
+    if fs.isDir( sBasePath ) then
+        if tStartups == nil then
+            tStartups = {}
+        end
+        for _,v in pairs( fs.list( sBasePath ) ) do
+            tStartups[ #tStartups + 1 ] = shell.resolveProgram( "/" .. fs.combine( sBasePath, v ) )
```

Comment:
I don't really see the point of using `shell.resolveProgram` here, you already have the directory path and filename so you may as well just use `fs.combine(sBasePath, v)`